src/clr-addons/combobox/combobox.ts

```diff
@@ -24,12 +25,16 @@ import {
 import {
   ClrLabel,
   ClrPopoverToggleService,
+  ɵb as RootDropdownService,
   ɵba as LayoutService,
   ɵbb as NgControlService,
   ɵbd as IfErrorService,
   ɵbc as ControlClassService,
   ɵe as POPOVER_HOST_ANCHOR,
   ɵz as ControlIdService,
+  ɵj as DROPDOWN_FOCUS_HANDLER_PROVIDER,
```

Comment:
pls add all of those imports to the tests, so we can be sure that we import the correct stuff (also the ones from options.ts